lib/workers/global/index.ts

```diff
@@ -38,6 +38,10 @@ export async function getRepositoryConfig(
     globalConfig,
     is.string(repository) ? { repository } : repository,
   );
+  const repoParts = repoConfig.repository.split('/');
```

Comment:
As said in the other PR, this would be a prime candidate for extraction to an utility